core/pom.xml

```diff
@@ -118,6 +118,7 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
+            <version>7.7.0-901-ce</version>
```

Comment:
please don't use fixed version

Reply:
This is due to local testing - I had to generate the artefacts locally as the DefaultSslEngineFactoy change PR is still not merged